src/java.base/share/classes/jdk/internal/loader/NativeLibraries.java

```diff
@@ -54,16 +55,14 @@
  * has already been loaded by a class loader with another class loader
  * will fail.
  */
-public final class NativeLibraries {
+public abstract class NativeLibraries {
```

Comment:
could this be sealed with only two specific subtypes defined here?

Reply:
It could.   It's just an internal class and not so much of a concern of unexpected subclasses.  I'll leave it as is.